test/Interpreter/protocol_extensions.swift

```diff
@@ -1,6 +1,11 @@
-// RUN: %target-run-simple-swift | %FileCheck %s
+// RUN: %target-run-simple-swift
```

Comment:
Once upon a time these were deliberately minimal tests, where if something broke in StdlibUnittest it wouldn't automatically break in these tests. Maybe that's not so relevant now, though.

Reply:
Yeah, I understand the reasoning here, but I find StdlibUnittest easier to work with when a test fails, and also CHECK: lines are not totally reliable because it's easy to accidentally match the wrong thing. At one point Dmitri told me that new executable tests should use StdlibUnittest so I've been slowly converting them when I touch them.